packages/ckeditor5-engine/src/conversion/upcasthelpers.js

```diff
@@ -330,8 +335,61 @@ export default class UpcastHelpers extends ConversionHelpers {
 	 * @returns {module:engine/conversion/upcasthelpers~UpcastHelpers}
 	 */
 	elementToMarker( config ) {
+		console.warn(
+			'upcast-helpers-element-to-marker-deprecated: ' +
+			'The UpcastHelpers#elementToMarker() method has been deprecated and will be removed in the near future. ' +
+			'Please use #dataToMarker() method instead.' );
+
 		return this.add( upcastElementToMarker( config ) );
 	}
+
+	/**
+	 * View to model marker conversion helper.
+	 *
+	 * Converts view data created by {@link module:engine/conversion/downcasthelpers~DowncastHelpers#markerToData} back to a model marker.
```

Comment:
```suggestion
	 * Converts view data created by {@link module:engine/conversion/downcasthelpers~DowncastHelpers#markerToData `#markerToData()`} back to a model marker.
```

Reply:
Otherwise, it looks like this:

![image](https://user-images.githubusercontent.com/156149/87926775-78d4e980-ca82-11ea-8c61-577ed735045e.png)